packages/cf-component-pagination/src/PaginationLink.js

```diff
@@ -1,74 +0,0 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { createComponent } from 'cf-style-container';
-
-const styles = ({ theme, disabled, active, type: linkType }) => ({
-  position: theme.position,
-  display: theme.display,
-  paddingTop: theme.paddingTop,
-  paddingBottom: theme.paddingBottom,
-  paddingLeft: theme.paddingLeft,
-  paddingRight: theme.paddingRight,
-  textDecoration: theme.textDecoration,
-  fontWeight: theme.fontWeight,
-  color: theme.color,
-  ':focus': {
-    zIndex: theme['zIndex:focus']
-  },
-  cursor: (disabled && theme.cursorDisabled) || (active && theme.cursorActive),
-  ...theme[linkType]
-});
-
-class PaginationLink extends Component {
-  constructor(props) {
-    super(props);
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  handleClick(e) {
-    e.preventDefault();
-    this.props.onClick();
-  }
-
-  render() {
-    const { clickable, label, children, className } = this.props;
-
-    if (clickable) {
-      return (
-        <a
-          className={className}
-          href="#"
-          onClick={this.handleClick}
```

Comment:
I was thinking that instead of handling this for the user, they would take in the event in their click handler in case they want to do something with the events themselves.